### ISSUE TITLE
Default arguments logging is confusing

### ISSUE THREAD

**Issue by RMeli** [RMeli/gnina-torch#21]
The function
https://github.com/RMeli/gnina-torch/blob/f8aecb6688ee359ee9499f0cb3019094ec7058a0/gnina/utils.py#L9-L33
is too simplistic. This function prints out all the CLI arguments to the log file, including the default ones. 

This might be confusing at times. For example, when there is no binding affinity annotation the following is printed to the log file:
```text
[...]
affinity_pos = None
[...]
pseudo_huber_affinity_loss = False
delta_affinity_loss: 4.00000E+00
scale_affinity_loss: 1.00000E+00
penalty_affinity_loss: 1.00000E+00
[...]
```

Only the variables that are relevant for the run should be printed out. In this case `affinity_pos = None` should be printed (to indicate that there is no binding affinity annotation), but everything else is superfluous and confusing.

The same happens for dynamic learning rate:
```text
[...]
lr_dynamic = False
lr_patience = 5
lr_reduce: 1.00000E-01
lr_min: 1.00000E-05
[...]
```
`lr_dynamic = False` should be printed, everything else is superfluous and confusing.
